### DISCUSSION TITLE
Show running balance

### DISCUSSION THREAD

**Discussion by Tuxie** [firefly-iii/firefly-iii#5244] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been discussed before.

### Description

I imported a few thousand transactions from YNAB into an empty Firefly III installation, and while it looks like the import was successful at a first glance, the account balances are _way_ off. I want to go through them to fix what went wrong, but comparing the transaction in Firefly III with the transactions in YNAB is very hard without being able to see the running balance.

### Solution

Have a balance column in the transactions list to see what the account balance was at that point in time.

**Comment by death2cupbots** [comment]
You can get a running balance if you do a transaction history overview (audit) from the reports page.

**Comment by JC5** [comment]
Indeed. This is on my list for the new layout.

**Reply by nomandera** [reply]
Most (all?) of my imports have a running balance field. Current I do not import this data but there is probably value in having an "balance according to bank" field. Being able to look back and see exactly where balances diverge for reconciliation would be a powerful quality of life feature.

**Comment by Tuxie** [comment]
Thank you! I found the problem, and it was that all transfers between accounts were duplicated in the import. I guess they were imported once from the source account and once from the target account. There already is an open bug report for this (#4493) so I'll just clean it up manually.
